src/test/convert-to-single-host.ts

```diff
@@ -46,7 +46,7 @@ describe('Office-Add-Taskpane-Ts projects', () => {
 
     describe('Office-Add-Taskpane project', () => {
         before((done) => {
-            helpers.run(path.join(__dirname, '../app')).withPrompts(answers).on('end', done);
+            helpers.run(path.join(__dirname, '../app')).withOptions({ 'prerelease': true }).withPrompts(answers).on('end', done);
```

Comment:
It would be nice if you could check whether the test is called with a prerelease option, e.g. `npm run test -- --prerelease` or just the regular `npm run test` and if --prerelease is an argument it could initialize the options as `{ 'prerelease': true}` or just `{ }`. I'm not sure if that is easy to do, but this would give the flexibility to have the tests be able to run against either without having it hardcoded.

Reply:
yeah, I think this is do-able